### ISSUE TITLE
Ability to persist to custom datastore

### ISSUE THREAD

**Issue by ghuser** [egoist/codepan#37]
You can use an abstraction layer (e.g. https://github.com/1602/jugglingdb) to provide the feature of persisting anonymous pans to a custom datastore, just by providing datastoretype/schema/port/credentials.

**Comment by egoist** [comment]
Thanks for the recommendation, I know I can but we don't have a backend

**Comment by ghuser** [comment]
I guess you've already written some code for persisting changes as gist. This is already a backend.

**Comment by egoist** [comment]
Yeah, we can use some other public APIs to store data, but not a database orm I guess.

**Comment by ghuser** [comment]
..yet
